packages/client/src/components/common/modals/balance-charge-modal.tsx

```
@@ -0,0 +1,388 @@
+      );
+    }
+  }, [financialEntitiesData, setFinancialEntities]);
+
+  const { generateBalanceCharge } = useGenerateBalanceCharge();
+
+  const onSubmit: SubmitHandler<z.infer<typeof formSchema>> = useCallback(
+    async data => {
+      try {
+        await generateBalanceCharge(data as GenerateBalanceChargeMutationVariables);
+        onOpenChange(false);
+      } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)